skiko/src/awtMain/objectiveC/macos/DisplayLinkThrottler.mm

```diff
@@ -65,6 +89,11 @@ - (void)invalidateDisplayLink {
 }
 
 - (void)setupDisplayLinkForWindow:(NSWindow *)window {
+    if (_isSleeping) {
+        /// System is sleeping, don't setup display link
```

Comment:
Should we nullify the display link?

Reply:
No, it's `nil` at this point, because when _isSleeping is set to true, display link is also deinitialised.